pyccl/core.py

```diff
@@ -789,7 +857,25 @@ def compute_nonlin_power(self):
 
     def compute_sigma(self):
         """Compute the sigma(M) and mass function splines."""
+        if self.has_sigma:
+            return
+
         # we need these things before building the mass function splines
+        if np.sum(self._params_init_kwargs['m_nu']) > 0:
```

Comment:
This combination has come up enough - and is useful enough to users - that I'm wondering if there should be a (presumably read-only) property for it?

Reply:
Good idea! I've switched to use the number of massive neutrinos that CCL computes internally.